Utils/utils.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+"""
+Utils module
+
+This module is holds the common logic across the codebase
+"""
+import logging
+
+
+class Utils:
+    @classmethod
+    def get_insecure_warning(cls):
+        logging.debug("Warning: Running the command with '--insecure' makes your system vulnerable to security attacks")
```

Comment:
maybe set up a static boolean CHANGED=FALSE so this will not be called every time we use this function. only the first time

Reply:
What do you mean? We do need it everytime a user runs the CLI.

Moving it to the client.py so that it will be the first thing the user sees because client initialization happens before the command is run